TenureInformationApi/V1/Boundary/Requests/CreateTenureRequestObject.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TenureInformationApi.V1.Domain;
+
+namespace TenureInformationApi.V1.Boundary.Requests
+{
+    public class CreateTenureRequestObject
+    {
+        public Guid Id { get; set; }
+        public string PaymentReference { get; set; }
+        public List<HouseholdMembers> HouseholdMembers { get; set; }
+        public TenuredAsset TenuredAsset { get; set; }
+        public Charges Charges { get; set; }
+        public DateTime StartOfTenureDate { get; set; }
+        public DateTime? EndOfTenureDate { get; set; }
+        public TenureType TenureType { get; set; }
+        public bool IsActive { get; set; }
```

Comment:
Should `IsActive` be here (and in swagger) given that it is a field calculated from the `EndOfTenureDate` value?

Reply:
I've removed IsActive as we don't need it